sdk/communication/communication-phone-numbers/test/public/lro.update.spec.ts

```diff
@@ -37,7 +41,23 @@ matrix([[true, false]], async function(useAad) {
     });
 
     it("can update a phone number's capabilities", async function() {
-      const update: PhoneNumberCapabilitiesRequest = { calling: "none", sms: "outbound" };
+      let callingCapabilityType: PhoneNumberCapabilityType = "outbound";
+      let smsCapabilityType: PhoneNumberCapabilityType = "inbound+outbound";
+      const currentPhoneNumber = await client.getPurchasedPhoneNumber(purchasedPhoneNumber);
+
+      if (currentPhoneNumber.capabilities.calling === callingCapabilityType) {
+        callingCapabilityType = "inbound";
```

Comment:
I simplified this to something like: 
`let callingCapabilityType: PhoneNumberCapabilityType = currentPhoneNumber.capabilities.calling === "outbound" ? "inbound" : "outbound" `